.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,21 +42,16 @@ Documentation change PR (review please): https://github.com/status-im/status.im/
 - network consumption
 
 ### Steps to test
-<!-- (Specify exact steps to test if there are such) -->
+[comment]: #  (Specify exact steps to test if there are such)
 
 - Open Status
 - ...
 - Step 3, etc.
 
-<!-- (PRs will only be accepted if squashed into single commit.) -->
-
-### Before and after screenshots comparison
```

Comment:
Never seen this one used. Usually screenshots are attached in summary

Reply:
This was used a couple times, for example I'm pretty sure I've seen in PRs for quo components. But fair enough to remove 👍🏼 I have a macro to generate an HTML table for before/after screenshots. At one point in Status I used this macro a lot, but never this part from the template.